corehq/apps/userreports/pillow.py

```diff
@@ -27,6 +28,23 @@
 REBUILD_CHECK_INTERVAL = 60 * 60  # in seconds
 
 
+def time_ucr_process_change(method):
+    def timed(*args, **kw):
+        ts = datetime.now()
+        result = method(*args, **kw)
+        te = datetime.now()
+        seconds = (te - ts).total_seconds()
+        if seconds > 0.1:
+            table = args[1]
+            doc = args[2]
+            message = u"UCR data source {} on doc_id {} took {} seconds to process".format(
+                table.config._id, doc['_id'], seconds
+            )
+            pillow_logging.warning(message)
```

Comment:
Where can we see pillow warning logs? The logging level for pillowtop is `ERROR`